source/val/validate_builtins.cpp

```diff
@@ -667,6 +667,36 @@ class BuiltInsValidator {
   // instruction.
   void Update(const Instruction& inst);
 
+  uint32_t GetMeshEntryPoint() {
```

Comment:
Can you only have a single mesh entry point in a module? I'm confused why this is a single entry point.